pkg/operator/operatorclient/common.go

```diff
@@ -0,0 +1,21 @@
+package operatorclient
+
+import (
+	"context"
+	"fmt"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+	"time"
+)
+
+func AwaitInformerCacheSync(name string, inf cache.SharedIndexInformer) error {
```

Comment:
not the best place to put, but I couldn't put it into `ceohelpers` because of cyclic imports :/ it wasn't that useful to create a new package for it either